doc: adding documentation to know how to work on the project (#165)
vuemontreal/vuemontreal: docs/workflow.md

# Workflow

## Branch naming

Please follow this naming pattern:

- feature => f/#issuenumer-small-description
- issue => b/#issuenumber-small-description

## Release

> We are working with github projects. Our next branch will always the next release incoming.

Reviewer: ```suggestion
We are working with github projects. Our `next` branch will always be the upcoming release.
```

## Process

> When you will create your merge request. Your branch will have to validate these conditions:

Reviewer: ```suggestion
When you create your merge request, your branch will have to match the following conditions:
```

- Respect the [ISSUE_TEMPLATE](../.github/ISSUE_TEMPLATE/bug_report.md) or [FEATURE_TEMPLATE](../.github/ISSUE_TEMPLATE/feature_request.md)

> - Have a coherent branch name

Reviewer: ```suggestion
- Have a coherent branch name (See [Branch naming](#branch-naming))
```

- Validate eslint

> - Validate our chromatic visual non regression

Reviewer 2: gonna need more explanations about this step

Author: i wanted to give the basic process from the starting point to the pull request. What do yout hink can be the best here ?
